route/service.go

```diff
@@ -0,0 +1,55 @@
+package route
+
+import (
+	"fmt"
+	"github.com/opentracing/opentracing-go/log"
+	"github.com/thunderdb/ThunderDB/consistent"
+	"github.com/thunderdb/ThunderDB/proto"
+)
+
+type DhtService struct {
+	thisNode proto.Node
+	hashRing *consistent.Consistent
+}
+
+func NewDhtService() *DhtService {
+	return &DhtService{
+		hashRing: consistent.New(),
+	}
+}
+
+type GetNeighborsReq struct {
+	NodeId  proto.NodeId
+	Count   int
+	Version string
+}
+
+type GetNeighborsResp struct {
+	Nodes   []proto.Node
+	ErrMsg  string
+	Version string
+}
+
+func (dht *DhtService) GetNeighbors(req *GetNeighborsReq, resp *GetNeighborsResp) (err error) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method DhtService.GetNeighbors should have comment or be unexported